.gitignore

```diff
@@ -114,3 +121,4 @@ dist
 .yarn/build-state.yml
 .yarn/install-state.gz
 .pnp.*
+>>>>>>> e76a839d74daa7193f44cd49ee0cba17b5bac1df
```

Comment:
remove this line - That's how you merge an individual file when there are merge conflicts when you want them all. See https://opensource.com/article/20/4/git-merge-conflict for more details